README.md

```diff
@@ -1,59 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# MyPortFolio Project
 
-> One paragraph statement about the project.
+> This project is based on my portfolio and is made with help of figma,html and css.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+It contains links to my personal pages.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
**Optional**l

- [ ] I will recommend you add `Getting started` section to the readme. Highlighting instructions on how to setup and run the project locally.